app/package.json

```diff
@@ -3,46 +3,63 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "next build",
-    "dev": "next dev",
+    "build": "NEXT_TELEMETRY_DISABLED=1 next build",
+    "dev": "NEXT_TELEMETRY_DISABLED=1 next dev",
     "format": "prettier --write '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
     "format-check": "prettier --check '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
-    "lint": "next lint",
-    "start": "next start",
+    "lint": "NEXT_TELEMETRY_DISABLED=1 next lint",
+    "postinstall": "mkdir -p ./public && cp -R ./node_modules/@uswds/uswds/dist/ ./public/uswds/",
+    "start": "NEXT_TELEMETRY_DISABLED=1 next start",
+    "storybook": "start-storybook -p 6006",
+    "storybook-build": "build-storybook",
     "test": "jest --ci --coverage",
     "ts:check": "tsc --noEmit"
   },
   "dependencies": {
-    "@types/jest-axe": "^3.5.3",
-    "@typescript-eslint/parser": "^5.26.0",
-    "jest-axe": "^6.0.0",
-    "next": "12.1.6",
-    "react": "18.1.0",
-    "react-dom": "18.1.0"
+    "@uswds/uswds": "^3.0.1",
+    "next": "^12.1.6",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "sass": "^1.52.3"
```

Comment:
nit: I think `sass` would be a dev dependency

Reply:
Awesome. Thank you.